*** /opt/rt5/share/html/Elements/ShowTransaction     Tue Sep  1 07:24:10 2020
--- /opt/rt5/share/html/Elements/ShowTransactionNew  Tue Sep  1 07:32:48 2020
***************
*** 180,185 ****
--- 180,186 ----
          && ( $can_modify
              || $Object->CurrentUserHasRight( $has_right{'ReplyTo'} )
          )
+       && $txn_type ne 'Comment'
      ) {
          my $alt = loc('Reply');
          push @actions, {
